src/main/io/serial.c

```diff
@@ -156,17 +162,24 @@ const char* serialPortNames[SERIAL_PORT_COUNT] = {
 #endif
 };
 
-const uint32_t baudRates[] = {0, 9600, 19200, 38400, 57600, 115200, 230400, 250000,
-        400000, 460800, 500000, 921600, 1000000, 1500000, 2000000, 2470000}; // see baudRate_e
+const uint32_t baudRates[] = {
+        0, 9600, 19200, 38400, 57600, 115200, 230400, 250000,
+        400000, 460800, 500000, 921600, 1000000, 1500000, 2000000, 2470000
+    }; // see baudRate_e
 
 static serialPortConfig_t* findInPortConfigs_identifier(const serialPortConfig_t cfgs[], size_t count, serialPortIdentifier_e identifier)
 {
+    if (identifier == SERIAL_PORT_NONE || identifier == SERIAL_PORT_ALL) {
```

Comment:
(SERIAL_PORT_ALL is not strictly necessary - it is never stored in cfgs[i].identifier )

Reply:
Whilst that is true currently, someone may inadvertently set it - so this is an added margin of safety.